### DISCUSSION TITLE
useJWTtoken:undefined

### DISCUSSION THREAD

**Discussion by elttilz** [rnadigital/agentcloud#7] [Q&A]
<img width="261" alt="image" src="https:github.com/rnadigital/agentcloud/assets/95276937/f64eb50f-e22a-4e26-8034-603cae63d841">

Ran into this problem and then got stuck。 I see this option in the config file, but I don't know where to get this JWTtoken?

**Comment by kodiii** [comment]
i keep getting the same, can't figure it out

**Comment by chymian** [comment]
@elttilz @kodiii, did any of you find a solution to this?
I  set a secret in the compse file.
I have the pbl, after container restart, I cannot login with the user credentials anymore. is that related?

**Comment by anada10** [comment]
Hi all this issue has been resolved, please git pull latest and let us know if you run into any issues.
